src/zenml/client.py

```diff
@@ -1549,9 +1547,9 @@ def _validate_stack_configuration(self, stack: StackRequest) -> None:
                 )
 
                 configuration = validate_stack_component_config(
```

Comment:
This is being called here for the validation of custom flavors right? Or do we need it duplicated on the client side for some other reason?

Reply:
Yes.